src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_rds.json

```
@@ -28,7 +28,7 @@
     "path": "/ValueTypes/AWS::RDS::DBInstance.Engine",
     "value": {
       "AllowedPattern": "Has to be one of [aurora, aurora-mysql, aurora-postgresql, mariadb, mysql, oracle-ee, oracle-se2, oracle-se1, oracle-se, postgres, sqlserver-ee, sqlserver-se, sqlserver-ex, sqlserver-web]",
-      "AllowedPatternRegex": "(?i)(aurora|aurora-mysql|aurora-postgresql|mariadb|mysql|oracle-ee|oracle-se2|oracle-se1|oracle-se|postgres|sqlserver-ee|sqlserver-se|sqlserver-ex|sqlserver-web)$"
+      "AllowedPatternRegex": "(?i)(aurora|aurora-mysql|aurora-postgresql|mariadb|mysql|oracle-ee|oracle-se2|oracle-se1|oracle-se|postgres|sqlserver-ee|sqlserver-se|sqlserver-ex|sqlserver-web|custom-sqlserver-web|custom-sqlserver-se|custom-sqlserver-ee)$"
```

seeing references for `custom-oracle-ee` as well
```suggestion
      "AllowedPatternRegex": "(?i)(custom-)?(aurora|aurora-mysql|aurora-postgresql|mariadb|mysql|oracle-ee|oracle-se2|oracle-se1|oracle-se|postgres|sqlserver-ee|sqlserver-se|sqlserver-ex|sqlserver-web)$"
```

---

you can get list of all supported engines via AWS cli/api - `aws rds describe-db-engine-versions --output json`